packages/evm/test/debugOutput.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { Chain, Common, Hardfork } from '@ethereumjs/common'
+import debug from 'debug'
+import * as tape from 'tape'
+
+import { EVM } from '../src'
+
+import { getEEI } from './utils'
+
+tape('Tests EVM constructor option to disable command line debugging output', (test) => {
+  const it = test.test
+  it('Should default to DEBUG: true', async (t) => {
+    debug.enable('evm')
+    const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Constantinople })
+    const eei = await getEEI()
+    const evm = await EVM.create({ eei, common })
+    t.ok(evm.DEBUG, 'DEBUG defaults to true')
+    t.end()
+  })
+  it('Should set DEBUG: false from `deactivateCLDebug: true` constructor option', async (t) => {
+    const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Constantinople })
+    const eei = await getEEI()
+    const evm = await EVM.create({ eei, common, deactivateCLDebug: true })
+    t.notok(evm.DEBUG, 'DEBUG set to false')
+    t.end()
+  })
+  test.end()
```

Comment:
```suggestion
tape('Tests EVM constructor option to disable command line debugging output', (t) => {
  t.test('Should default to DEBUG: true', async (st) => {
    debug.enable('evm')
    const common = new Common({ chain: Chain.Mainnet })
    const eei = await getEEI()
    const evm = await EVM.create({ eei, common })
    st.ok(evm.DEBUG, 'DEBUG defaults to true')
    st.end()
  })
  it('Should set DEBUG: false from `deactivateCLDebug: true` constructor option', async (st) => {
    const common = new Common({ chain: Chain.Mainnet })
    const eei = await getEEI()
    const evm = await EVM.create({ eei, common, deactivateCLDebug: true })
    st.notok(evm.DEBUG, 'DEBUG set to false')
    st.end()
  })
  t.end()
```
Current version is fine, just updated for style consistency with the rest of our tests. Also removing the hardfork common option since it's not necessary.